lib/Dialect/RTG/Transforms/ElaborationPass.cpp

```diff
@@ -1026,6 +1109,7 @@ class Elaborator : public RTGOpVisitor<Elaborator, FailureOr<DeletionKind>> {
   DenseMap<Value, ElaboratorValue> state;
 
   uint64_t virtualRegisterID = 0;
+  uint64_t uniqueLabelID = 1;
```

Comment:
Move to shared state.

Reply:
Similar to VirtualRegisters, UniqueLabels could rely on a malloc'd address instead giving each one a unique ID.